controllers/metal3.io/baremetalhost_controller.go

```diff
@@ -1128,8 +1129,14 @@ func (r *BareMetalHostReconciler) actionPreparing(prov provisioner.Provisioner,
 	hfsDirty, hfs, err := r.getHostFirmwareSettings(info)
 
 	if err != nil {
-		// wait until hostFirmwareSettings are ready
-		return actionContinue{subResourceNotReadyRetryDelay}
+		switch {
+		case errors.As(err, &ConfigurationError{}):
+			// wait for the user to correct the error
+			return actionContinue{configurationErrRetryDelay}
```

Comment:
Why is it different from what we had before? You're only changing the retry delay for 1 to 1.5 minutes. Did you intend to you something else than actionContinue?

Reply:
Previously, the operation did not return an error, and there was no retry delay (see line 1892 below).

Reply:
I think on line 1132 the error would be returned in any case.